resources/migrations/001_update_migrations.yaml

```
@@ -9270,6 +9270,119 @@ databaseChangeLog:
                   type: ${text.type}
                   remarks: running, failed, or completed
 
+  - changeSet:
+      id: v51.2024-09-05T08:00:00
+      author: qnkhuat
+      comment: Create the notification table
+      preConditions:
+        - not:
+          - tableExists:
+              tableName: notification
+      changes:
+        - createTable:
+            tableName: notification
+            remarks: join table that connect notification subscriptions and notification destinations
```

Tracking our rename, also there's no need to repeat the qualifier.

```suggestion
            remarks: join table that connect notification subscriptions and handlers
```